colabs/huggingface/Huggingface_wandb.ipynb

```diff
@@ -199,12 +191,39 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Model Artifacts\n",
     "\n",
-    "### 📈 Track key information effortlessly by default\n",
+    "You will also be able to view your model checkpoints in the W&B [Artifacts](https://docs.wandb.ai/guides/artifacts) page of the run above. These artifacts store all the metadata associated with this model, the W&B runs consuming it, and the whole lineage of upstream and downstream artifacts!\n",
+    "\n",
+    "Here's an example of the artifacts generated from the above run\n",
+    "\n",
+    "![run_artifacts](https://drive.google.com/uc?export=view&id=1janWuNXhxgIbNnSZjSON51dq-DVv8S0i)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model Registry\n",
+    "After logging a bunch of checkpoints across multiple runs during experimentation, now comes time to hand-off the best checkpoint to the next stage of the workflow (e.g. testing, deployment).\n",
+    "\n",
+    "The Model Registry is a central page that lives above individual W&B projects. It houses **Registered Models**, portfolios that store \"links\" to the valuable checkpoints living in individual W&B Projects.\n",
+    "\n",
+    "The model registry offers a centralized place to house the best checkpoints for all your model tasks. Any `model` artifact you log can be \"linked\" to a Registered Model. Please refer to the following [tutorial](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb-model-registry/Model_Registry_E2E.ipynb) for more information related to the model registry\n",
```

Comment:
Maybe we should link the Model Registry docs rather than jumping straight into the colab? https://docs.wandb.ai/guides/models:

`Pleaser refer to [this](https://docs.wandb.ai/guides/models) guide where you can find a [walkthrough](https://docs.wandb.ai/guides/models/walkthrough) of the Model Registry to learn more. `

Reply:
Updated as suggested.